hadoop-hdds/common/dev-support/findbugsExcludeFile.xml

```diff
@@ -31,6 +31,10 @@
     <Method name="update" />
     <Bug pattern="SF_SWITCH_FALLTHROUGH,SF_SWITCH_NO_DEFAULT" />
   </Match>
+  <Match>
+    <Class name="org.apache.hadoop.hdds.utils.db.CodecBufferr" />
```

Comment:
```suggestion
    <Class name="org.apache.hadoop.hdds.utils.db.CodecBuffer" />
```

Reply:
On second thought, let's not suppress the warning, I'll post a PR to fix the warning, even if it's false positive.

Reply:
#5638